conda/pygpu/meta.yaml

```diff
@@ -13,9 +13,6 @@ build:
 
 requirements:
   build:
-    - m2-git         [win]
```

Comment:
why this fix the problem?

Also, we need to make the build fail if it can't generate a good version.

Reply:
Because it is that git that doesn't like --dirty and gets used for the setup.py version.

Also we don't need it since we grab the source from the existing checkout.